components/Article.jsx

```diff
@@ -0,0 +1,17 @@
+import { getArticles } from "../api";
```

Comment:
The singular name of this Component could be confusing when it comes to adding the single article page.